site/content/docs/0.0/utilities/shadows.md

```diff
@@ -18,25 +18,25 @@ Box-shadow styles are referred to as 'elevation' within the design system.
 You can <!--also -->quickly add or remove a shadow with our `box-shadow` utility classes. Includes support for `.shadow-none` and new semantic helpers.
 
 {{< example class="overflow-hidden" >}}
-<div class="shadow-none p-tall mb-huge bg-body-tertiary">No shadow</div>
-<div class="shadow-default p-tall mb-huge bg-body-tertiary">Default shadow</div>
-<div class="shadow-drag p-tall mb-huge bg-body-tertiary">Drag shadow</div>
-<div class="shadow-emphasized p-tall mb-huge bg-body-tertiary">Emphasized shadow</div>
-<div class="shadow-raised p-tall mb-huge bg-body-tertiary">Raised shadow</div>
-<div class="shadow-sticky-default p-tall mb-huge bg-body-tertiary">Sticky default shadow</div>
-<div class="shadow-sticky-emphasized p-tall mb-huge bg-body-tertiary">Sticky emphasized shadow</div>
-<div class="shadow-sticky-navigation-scrolled p-tall mb-huge bg-body-tertiary">Sticky navigation shadow</div>
+<div class="shadow-none p-tall mb-huge bg-secondary">No shadow</div>
```

Comment:
the shadow is not there anymore since the variable name has changed. I remember we talked about this token name change but I don't know if we acted where we should fix that ?!

Reply:
Ohh, good catch, I think that we should have done it in #2816. Are you fine if it's inside this PR ? I totally forgot about those one. Thanks 🙏

Reply:
I made a change a the color branch directly. In https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2802/commits/0fc5dd46d1b6fca187f012f466c8c88641662012.